pkgs/games/chocolate-doom/default.nix

```diff
@@ -11,12 +11,15 @@ stdenv.mkDerivation rec {
     sha256 = "0ajzb767wyj8vzhjpsmgslw42b0155ji4alk26shxl7k5ijbzn0j";
   };
 
+  patches = [ ./CVE-2020-14983.patch ];
```

Comment:
After #92130 only the `ticdup` patch should be necessary (is that also a CVE?). Please use `fetchpatch` to get it from the upstream repository:

https://github.com/chocolate-doom/chocolate-doom/commit/54fb12eeaa7d527defbe65e7e00e37d5feb7c597.diff

Reply:
I'm not sure honestly. When upstream linked to the fix for CVE-2020-14983 they included it, so I included it.